neo/Network/P2P/Payloads/Block.cs

```diff
@@ -131,5 +134,31 @@ public TrimmedBlock Trim()
                 ConsensusData = ConsensusData
             };
         }
+
+        public StackItem ToStackItem()
+        {
+            return new VM.Types.Array
+            (
+                new StackItem[]
+                {
+                    // Base
+                    new Integer(Version),
+                    new ByteArray(PrevHash.ToArray()),
+                    new ByteArray(MerkleRoot.ToArray()),
+                    new Integer(Timestamp),
+                    new Integer(Index),
+                    new ByteArray(NextConsensus.ToArray()),
+                    new Integer(Version),
+                    // Witness
+
+                    // Count
+                    // ConsensusData
+                    // Transactions
+                    new VM.Types.Array(Transactions.Select(u=>u.ToStackItem())),
```

Comment:
I think the interoperable block shouldn't include the transactions. Otherwise, it is very likely that the stack size will exceed the limit of 1024.

Reply:
Only the hashes?